### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by marekrzytki**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml:15]: 1. `name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}"` -> There should be also info about Sylius version
2. There should be Sylius version and Symfony version in the matrix. Keep in mind we support both 1.9 and 1.10
3. We can bump node to 14.x
4. When you finalize the point 2 remember to add restricting Sylius and Symfony versions

As a reference you can use build from e.g. CMS plugin

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (marekrzytki) [reply .github/workflows/build.yml:15]: I can bump node version. Can I add 1 & 2 with Sylius 1.11 update?

File: spec/EventListener/ShippingExportEventListenerSpec.php [hunk 2]
```diff
@@ -29,49 +30,72 @@ function it_is_initializable(): void
...
-        $this->beConstructedWith($webClient, $soapClient);
+        $this->beConstructedWith(
+            $filesystem,
+            $shippingExportRepository,
+            'shippingLabel',
+             $shippingLabelFetcher
+        );
     }
 
     function it_export_shipment(
```
Comment (senghe) [inline spec/EventListener/ShippingExportEventListenerSpec.php]:
```suggestion
    function it_exports_shipment(
```

**Review by senghe — APPROVED** [review]
Approve in advance, but could you change the literal fix?

**Pull request merged**